src/CMakeLists.txt

```diff
@@ -134,6 +134,13 @@ else()
 endif()
 
 if(USE_LV2)
+    if(MSVC)
+        # symlinks require development mode on Windows machines...
```

Comment:
Just a bit of a clarification for the comment... what do you mean by `development mode`.  Last I tried, it needed an elevated environment (e.g. Run As Administrator, Windows equivalent of `sudo`).  The comments here confirm this: https://stackoverflow.com/a/61244115/3196753.

Reply:
I  followed a link to [the CMake PR enabling it on Windows](https://gitlab.kitware.com/cmake/cmake/-/merge_requests/2144). However, it seems multiple users have issues. Maybe I'll just replace it with "symlinks require special privilege on Windows"? :laughing: